runtime/oti/j9consts.h

```diff
@@ -342,6 +342,7 @@ extern "C" {
 #define J9_EXTENDED_RUNTIME2_LOAD_HEALTHCENTER_MODULE 0x4000
 #define J9_EXTENDED_RUNTIME2_3164_INTEROPERABILITY 0x8000
 #define J9_EXTENDED_RUNTIME2_ENABLE_CRIU_SUPPORT 0x10000
+#define J9_EXTENDED_RUNTIME2_CRIU_NON_PORTABLE_MODE 0x20000
```

Comment:
Do you still need this bit?  I think it can rolled into `vm->checkpointState` rather than using a (limited) extendedRuntimeFlags bit